### PR TITLE
OHARA-1473 Connector should not connect to a topic when entering a co…

### PR DESCRIPTION
**Pull request by eechih**
…nnector form the first time

1. 新加入的 Source Connector，預設不關聯任何 Topic
2. Source Connector 的 write topic 下拉選單，可以取消已選擇的 Topic
3. 取消已選擇的 Topic，自動更新 Pipeline graph，Source Connector 與 Topic 之間的線會消失

### CODE REVIEW

**Comment by konekoya** [comment]
retry manager

File: ohara-manager/client/src/components/pages/PipelinePage/connectors/FtpSource.js [hunk 1]
```diff
@@ -810,6 +796,8 @@ class FtpSource extends React.Component {
                     data-testid="write-topic-select"
                     handleChange={this.handleSelectChange}
                     disabled={isRunning}
+                    placeholder="Please select a topic"
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/PipelinePage/connectors/FtpSource.js]: Let's make the UI consistent, adding three dots at the end of the placeholder :)

`Please select a topic...`

File: ohara-manager/client/src/components/common/Form/Select.js [hunk 2]
```diff
@@ -69,9 +70,9 @@ const Select = ({
       height={height}
       {...rest}
     >
-      {placeholder && (
-        <option value="" disabled data-id="default-option">
-          {placeholder}
+      {(placeholder || clearable) && (
+        <option value="" disabled={!clearable} data-uuid="default-option">
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Form/Select.js]: Let's get rid of the uuid here and use id instead.

File: ohara-manager/client/src/components/pages/PipelinePage/connectors/JdbcSource.js [hunk 3]
```diff
@@ -560,7 +550,8 @@ class JdbcSource extends React.Component {
                 width="100%"
                 data-testid="write-topic-select"
                 handleChange={this.handleChangeSelect}
-                disabled={isRunning}
+                placeholder="Please select a topic"
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/PipelinePage/connectors/JdbcSource.js]: Same here. Please add three dots after the phase.

File: ohara-manager/client/src/components/pages/PipelinePage/connectors/FtpSource.js (hunk 1 above)
Comment (eechih) [reply ohara-manager/client/src/components/pages/PipelinePage/connectors/FtpSource.js]: ok

File: ohara-manager/client/src/components/pages/PipelinePage/connectors/JdbcSource.js (hunk 3 above)
Comment (eechih) [reply ohara-manager/client/src/components/pages/PipelinePage/connectors/JdbcSource.js]: ok

File: ohara-manager/client/src/components/common/Form/Select.js (hunk 2 above)
Comment (eechih) [reply ohara-manager/client/src/components/common/Form/Select.js]: Sorry~ I am often negligent. 😭

**Pull request merged**